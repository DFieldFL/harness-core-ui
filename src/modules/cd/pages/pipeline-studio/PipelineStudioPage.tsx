import React from 'react'
import { loggerFor, ModuleName } from 'framework'
import { Container, Heading } from '@wings-software/uikit'

const logger = loggerFor(ModuleName.CD, 'PipelineStudioPage')

export const PipelineStudioPage: React.FC = () => {
  logger.info('Mounting Pipeline Studio...')

  return (
    <Container padding="huge">
      <Heading>Pipeline Studio</Heading>
    </Container>
  )
}
