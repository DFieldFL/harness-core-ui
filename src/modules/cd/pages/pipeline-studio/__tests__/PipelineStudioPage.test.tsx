import React from 'react'
import { render } from '@testing-library/react'

import { PipelineStudioPage } from '../PipelineStudioPage'

describe('PipelineStudioPage tests', () => {
  test('page snapshot', () => {
    const { container } = render(<PipelineStudioPage />)
    expect(container).toMatchSnapshot()
  })
})
