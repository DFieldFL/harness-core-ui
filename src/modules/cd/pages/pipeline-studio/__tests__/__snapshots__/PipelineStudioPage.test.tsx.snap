// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PipelineStudioPage tests page snapshot 1`] = `
<div>
  <div
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---padding-huge--yMnr"
  >
    <h1
      class="StyledProps---font--2ipy StyledProps---font-h1--1R-x StyledProps---main--1Wws"
    >
      Pipeline Studio
    </h1>
  </div>
</div>
`;
